docs/intro_03_gnosis_safe.md

```diff
@@ -4,24 +4,24 @@ title: The Gnosis Safe
 sidebar_label: The Gnosis Safe
 ---
 
-The Gnosis Safe is a smart contracts wallet with the multi-signature functionality at its core. But this is not the only advantage the Gnosis Safe provides.
+The Gnosis Safe is a smart contract wallet with multi-signature functionality at its core. It enables the following features:
 
 ### High Security
 
 Gnosis Safe's **multi-signature** functionality allows you to define a list of owner accounts and a threshold number of accounts required to confirm a transaction. Once the threshold of owner accounts have confirmed a transaction, the Safe transaction can be executed. Owners can either be EOAs or other smart contract accounts.
 
-### Ether less accounts
-
-Another core functionality of the Gnosis Safe is **token payment**. Generally, Ethereum transactions require ETH for paying transaction fees (“gas”). With the Gnosis Safe, users can pay transaction fees in a number of supported ERC20 tokens. This is realized via a transaction relay service that accepts those tokens and submits the transactions to the blockchain, therefore paying the gas fee in ETH. 
-
 ### Advanced execution logic
 
 It is possible to make use of different **Gnosis Safe Library contracts** to perform complex transactions. A very common example of this are **batched transactions** where multiple simple Ethereum transactions are combined and executed at once. That means instead of having to sign several transactions sequentially, a user just needs to sign one batched transaction.
 
 ### Advanced access management
 
-By allowing to add **Gnosis Safe Modules** to a Gnosis Safe it is possible to defined fine-grained access management. It is for example possible to define a module that can only be use to **recover access** to a Gnosis Safe under specific circumstances. A very popular version of this is the **Social Recovery Module**
+You can add **Gnosis Safe Modules** to your Gnosis Safe. Thereby it is possible to implement more fine-grained access management. For instance, it is possible to define a module that can only be use to **recover access** to a Gnosis Safe under specific circumstances. A popular version of this is the **Social Recovery Module**. A different example are **allowance modules** which allows owners of a Gnosis Safe to grant limited exeuction permission such as a daily limit to external accounts.
```

Comment:
>A different example **are** ... which **allows** 

should they be either both plural or singular?

Reply:
yea, question is whether the allows refers to example or modules :)

I'll change it.